modules/querier/querier_query_range.go

```diff
@@ -122,67 +121,10 @@ func (q *Querier) queryBlock(ctx context.Context, req *tempopb.QueryRangeRequest
 	inspectedBytes, spansTotal, _ := eval.Metrics()
 
 	return &tempopb.QueryRangeResponse{
-		Series: queryRangeTraceQLToProto(res, req),
+		Series: res.ToProto(req),
```

Comment:
there are subtle differences between these two methods. are we sure they are equivalent?

Reply:
yeah, me and @mdisibio reviewed them today and res.ToProto is just better version of `queryRangeTraceQLToProto`.

res.ToProto has some extra checks and allocates less, other then that the logic is same.

feel free to audit it in case we missed something: https://github.com/grafana/tempo/blob/68786618c9bd6f986ce494d35b9c1dc84eaf5640/pkg/traceql/engine_metrics.go#L233-L304